Assets/Scripts/GenerateRandomPosition.cs

```diff
@@ -58,6 +58,12 @@ public Vector3 GenerateRandomPositionOnTerrain()
             GenerateRandomPositionOnTerrain();
         }
 
+        // Prevents default location from spawning plants
+        if (randomPosition.Equals(new Vector3(0, 0, 0)))
```

Comment:
This and the conditional on line 70, once uncommented and working, can be combined. In the future we should use the singleton to grab the location for the vector of the gameobjects. Then all we would do is check that the new vector isn't in that list or set.